cmd/root.go

```diff
@@ -144,17 +147,21 @@ GO_TEST_BINARY="gotest"
 			stdout, err := c.StdoutPipe()
 			check(err)
 
-			check(c.Start())
+			go func() {
```

Comment:
This would run all sub tests in parallel. Depending on how many packages you have it could spawn hundreds of goroutines that execute test packages. I think this should behave similar to how `go test -parallel` works. It's also important to have the option to disable parallelism, for example when you have database tests!